generate-release/README.md

```diff
@@ -0,0 +1,100 @@
+# Generate Release
+
+This CLI tool is used to generate all the skeleton files required to create a new release.
+
+For a bit more background see this issue: <https://github.com/bevyengine/bevy-website/issues/1163>
+
+All commands assume they are ran in the `./generate-release` folder.
+
+Each command will generate files in the `/release-content/{release-version}` folder. The `release-version` is an argument to all commands.
+
+Each command have a `--from` and `--to` argument. You can pass it a git tag or a git commit.
+
+Before running the command, you'll need to generate a github api token at <https://github.com/settings/tokens>. It's easeier to use classic tokens and you don't need any specific role, just the default is enough. Then add it to a file called `.env` like so:
+
+```env
+GITHUB_TOKEN=token_string_copied_from_github
+```
+
+Here's an example for the commands used to generate the 0.14 release:
+
+```shell
+cargo run -- --from v0.13.0 --to main --release-version 0.14 migration-guides
+cargo run -- --from v0.13.0 --to main --release-version 0.14 release-notes
+cargo run -- --from v0.13.0 --to main --release-version 0.14 changelog
+cargo run -- --from v0.13.0 --to main --release-version 0.14 contributors
+```
+
+## Migration Guides
+
+The `migration-guides` command will generate the `/release-content/{release-version}/migration-guides` folder.
+Inside will be a single `_guides.toml` that contains metadata needed for each guides. Then each guide will be a separate markdown file inside that folder.
+
+Once the files are generated, you can easily add a new migration guide by adding a new file in `content/learn/migration-guides`.
+
+Inside that file, you should have something that looks like this:
+
+```markdown
++++
+title = "0.13 to 0.14"
+insert_anchor_links = "right"
+[extra]
+weight = 9
+long_title = "Migration Guide: 0.13 to 0.14"
++++
+
+{% callout(type="warning") %}
+    Bevy relies heavily on improvements in the Rust language and compiler. As a result, the Minimum Supported Rust Version (MSRV) is "the latest stable release" of Rust.
+{% end %}
+
+{{ migration_guides(path="./release-content/0.14/migration-guides/") }}
+```
+
+The most important part of this is the `migrations_guides` shortcode. It will get the list of guides from the `_guides.toml` file and combine all the separate file and generate appropritate markup for it.
```

Comment:
```suggestion
The most important part of this is the `migrations_guides` shortcode. It will get the list of guides from the `_guides.toml` file and combine all the separate file and generate appropriate markup for it.
```